core/src/main/java/com/graphhopper/storage/CHStorageBuilder.java

```diff
@@ -63,9 +63,9 @@ public int addShortcutNodeBased(int a, int b, int accessFlags, double weight, in
 
     /**
      * @param origFirst The first original edge that is skipped by this shortcut. For example for the following shortcut
-     *                  edge from x to y, which itself skips the shortcuts x->v and v->y the first original edge would
-     *                  be x->u: x->u->v->w->y
-     * @param origLast  like origFirst, but the last orig edge, i.e w->y in above example
+     *                  edge from x to y, which itself skips the shortcuts x-&gt;v and v-&gt;y the first original edge would
+     *                  be x-&gt;u: x-&gt;u-&gt;v-&gt;w-&gt;y
+     * @param origLast  like origFirst, but the last orig edge, i.e w-&gt;y in above example
```

Comment:
This makes no sense to me to be honest. These comments are meant to be read when browsing the source code and using &gt; instead of > makes this unreadable. The javadocs as html pages on the other hand are used by literally nobody so I don't think we should modify the comments like this.

Reply:
I also don't understand why this is needed now when it was not needed for the 5.0 release?

Reply:
> I think the documentation inside the IDE parses it as html too (but might not be so strict). If it is just for source code then we could put it into /* normal non-javadoc comments.

Yes, maybe, but I checked and IntelliJ deals with the `>` symbol without problems for example. 

> If it is just for source code then we could put it into /* normal non-javadoc comments.

I like the code highlighting that is used for `/**`, but really I don't care as long as it is readable in IntelliJ and maybe Eclipse and Netbeans if anyone is using these. I definitely do not feel like trying to satisfy some html restrictions of some javadoc plugin that produces html that nobody needs :D

Reply:
Ok thanks, yes looks good to me.